### ISSUE TITLE
Clear Sky crashes on new game start

### ISSUE THREAD

**Issue by Blackfox-Thane** [Swartz27/xray-16#1517]
**Describe the bug**

I'm trying to play CS using openxray, but every time i start up a new game and select a difficulty it instantly crashes

**BugTrap error report**
> ! Couldn't create surface from image: Parameter 'src' is invalid
> OpenXRay Release Master Gold build 9046, Nov 13 2023 (Linux 64-bit, shared)
> Custom build from commit[481d4dc8edd58866a79e072dc65355259c08fe5b] branch[makepkg]
>  
> command line -cs 
> 
> * CPU features: RDTSC, MMX, SSE, AVX
> * CPU threads: 12
> 
> Initializing File System...
> $fs_root$ = /home/thane/.local/share/GSC Game World/S.T.A.L.K.E.R. - Clear Sky/
> using fs-ltx fsgame.ltx
> FS: 18352 files cached 28 archives, 28Kb memory used.
> Init FileSystem 0.088274 sec
> -----loading \home\thane\.local\share\GSC Game World\S.T.A.L.K.E.R. - Clear Sky\gamedata\configs\system.ltx
> -----loading \home\thane\.local\share\GSC Game World\S.T.A.L.K.E.R. - Clear Sky\gamedata\configs\system.ltx
> ! Failed to determine on which monitor the game is launched.
> ! SDL: Invalid window
> Starting INPUT device...
> Loading module: xrRender_GL
> Available render modes[1]:
> renderer_rgl
> Executing config-script "user.ltx"...
> [\home\thane\.local\share\GSC Game World\S.T.A.L.K.E.R. - Clear Sky\_appdata_\user.ltx] successfully loaded.
> Selected renderer: renderer_rgl
> Loading module: xrGame
> Initializing Engine...
> SOUND: OpenAL: system default sound device name is OpenAL Soft
> Executing config-script "\home\thane\.local\share\GSC Game World\S.T.A.L.K.E.R. - Clear Sky\gamedata\configs\default_controls.ltx"...
> [\home\thane\.local\share\GSC Game World\S.T.A.L.K.E.R. - Clear Sky\gamedata\configs\default_controls.ltx] successfully loaded.
> Executing config-script "user.ltx"...
> Executing config-script "\home\thane\.local\share\GSC Game World\S.T.A.L.K.E.R. - Clear Sky\gamedata\configs\rspec_default.ltx"...
> [\home\thane\.local\share\GSC Game World\S.T.A.L.K.E.R. - Clear Sky\gamedata\configs\rspec_default.ltx] successfully loaded.
> Executing config-script "\home\thane\.local\share\GSC Game World\S.T.A.L.K.E.R. - Clear Sky\gamedata\configs\default_controls.ltx"...
> [\home\thane\.local\share\GSC Game World\S.T.A.L.K.E.R. - Clear Sky\gamedata\configs\default_controls.ltx] successfully loaded.
> ~ Invalid syntax in call to 'sv_adm_menu_ban_time'
> ~ Valid arguments: ui_mp_am_10_minutes/ui_mp_am_30_minutes/ui_mp_am_1_hour/ui_mp_am_6_hours/ui_mp_am_1_day/ui_mp_am_1_week/ui_mp_am_1_month/ui_mp_am_3_monthes/ui_mp_am_forever
> ~ Invalid syntax in call to 'sv_suspicious_actions_ban_time'
> ~ Valid arguments: ui_mp_am_10_minutes/ui_mp_am_30_minutes/ui_mp_am_1_hour/ui_mp_am_6_hours/ui_mp_am_1_day/ui_mp_am_1_week/ui_mp_am_1_month/ui_mp_am_3_monthes/ui_mp_am_forever
> [\home\thane\.local\share\GSC Game World\S.T.A.L.K.E.R. - Clear Sky\_appdata_\user.ltx] successfully loaded.
> SOUND: Selected device is OpenAL Soft
> * SOUND: EFX extension: present
> * sound : cache: 65537 kb, 4856 lines, 13820 bpl
> Starting RENDER device...
> * GPU vendor: [NVIDIA Corporation] device: [NVIDIA GeForce RTX 2070 SUPER/PCIe/SSE2]
> * GPU OpenGL version: 4.1.0 NVIDIA 545.29.02
> * GPU OpenGL shading language version: 4.10 NVIDIA via Cg compiler
> * GPU OpenGL VTF units: [32] CTI units: [192]
> * DVB created: 4096K
> * DIB created: 512K
> ! Version conflict in shader 'def_shaders\def_vertex'
> ! Version conflict in shader 'default'
> ! Version conflict in shader 'effects\lightplanes'
> ! Renderer doesn't support blender 'effects\shadow_world'
> ! Version conflict in shader 'models\lightplanes'
> ! Version conflict in shader 'models\model'
> ! Version conflict in shader 'models\model_aref'
> ! Version conflict in shader 'models\model_fur'
> ! Version conflict in shader 'models\pautina'
> ! Version conflict in shader 'models\selflight'
> ! Version conflict in shader 'models\selflight_det'
> ! Version conflict in shader 'models\selflightl'
> ! Version conflict in shader 'models\transparent'
> ! Version conflict in shader 'models\xanomaly'
> ! Version conflict in shader 'models\xdistort'
> ! Version conflict in shader 'models\xdistortcolor'
> ! Version conflict in shader 'models\xdistortcolorl'
> ! Version conflict in shader 'models\xdistortinv'
> ! Version conflict in shader 'models\xmonolith'
> ! Version conflict in shader 'models\xwindows'
> ! Version conflict in shader 'selflight'
> * HWDST/PCF supported and used
> Starting engine...
> ! Can't find sound 'characters_voice\scenario\video\intro_gsc'
> ! Can't find sound 'characters_voice\scenario\video\ds_black'
> * [ Linux ]: free[11313824 K], reserved[188320 K], committed[4966544 K]
> * [ Render ]: textures[6825 K]
> * [ x-ray  ]: process heap[281 K]
> * [ x-ray  ]: economy: strings[8534 K], smem[0 K]
> * phase time: 0 ms
> * phase cmem: 303 K
> Loading sounds...
> Loading objects...
> Loading models...
> Loading textures...
> * [prefetch] time:   198 ms
> * [prefetch] memory: 73 Kb
> * phase time: 201 ms
> * phase cmem: 379 K
> ! [LUA][ERROR]  ERROR: There are no sound collection with path: characters_voice\scenario\agroprom\agr_base_commander_reward_pda
> stack traceback:
> 	...ld\S.T.A.L.K.E.R. - Clear Sky\gamedata\scripts\_g.script:974: in function 'abort'
> 	...L.K.E.R. - Clear Sky\gamedata\scripts\sound_theme.script:364: in function <...L.K.E.R. - Clear Sky\gamedata\scripts\sound_theme.script:326>
> 	[C]: in function 'actor_sound'
> 	...L.K.E.R. - Clear Sky\gamedata\scripts\sound_theme.script:774: in function 'load_sound'
> 	...ld\S.T.A.L.K.E.R. - Clear Sky\gamedata\scripts\_g.script:1148: in function <...ld\S.T.A.L.K.E.R. - Clear Sky\gamedata\scripts\_g.script:1077>
>  
> FATAL ERROR
>  
> [error] Expression    : 0
> [error] Function      : ErrorLog
> [error] File          : /home/thane/.cache/paru/clone/openxray/src/xray-16/src/xrScriptEngine/ScriptEngineScript.cpp
> [error] Line          : 40
> [error] Description   : ERROR: There are no sound collection with path: characters_voice\scenario\agroprom\agr_base_commander_reward_pda
>  
> 
> stack trace:
> 
> xrDebug::GatherInfo(char*, unsigned long, ErrorLocation const&, char const*, char const*, char const*, char const*)
> xrDebug::Fail(bool&, ErrorLocation const&, char const*, char const*, char const*, char const*)
> ErrorLog(char const*)
> luabind::detail::function_object_impl<void (*)(char const*), luabind::meta::type_list<void, char const*>, luabind::meta::type_list<> >::entry_point(lua_State*)
> /usr/lib/xrLuajit.so(+0x66675) [0x7f602350b675]
> luabind::detail::class_rep::constructor_dispatcher(lua_State*)
> /usr/lib/xrLuajit.so(+0x66675) [0x7f602350b675]
> /usr/lib/xrLuajit.so(lua_pcall+0xb3) [0x7f60234be883]
> luabind::detail::pcall(lua_State*, int, int)
> void luabind::call_function<void, luabind::meta::type_list<>>(luabind::adl::object const&)
> CALifeSimulator::CALifeSimulator(IPureServer*, shared_str*)
> game_sv_Single::Create(shared_str&)
> xrServer::Connect(shared_str&, GameDescriptionData&)
> CLevel::net_start2()
> CRenderDevice::BeforeFrame()
> CRenderDevice::ProcessFrame()
> CRenderDevice::message_loop()
> CRenderDevice::Run()
> Startup()
> RunApplication()

**Desktop**
 - OS: Artix Linux x86_64
 - OpenXRay Release Master Gold build 9046

**Comment by Xottab-DUTY** [comment]
Do you use GOG? In your game folder, please, rename `Localization` folder to `localization`.

**Comment by Blackfox-Thane** [comment]
Hello sorry for the late response just checked the folder it's definitely lowercased
![B](https://github.com/OpenXRay/xray-16/assets/107892323/1db272fc-eb60-46ed-9de2-f4b279f42721)

Maybe it's because i symlinked the files?
![A](https://github.com/OpenXRay/xray-16/assets/107892323/469dc347-f45a-45a6-a0ae-31f7079cf57d)

**Comment by Blackfox-Thane** [comment]
Hello again so I found out the issue to this problem was in the `user.itx` file. I had to manually change language line because it was still registering in Russian despite me having a `xenglish.db` file inside my `localization` folder. TY for the reply and sorry if i wasted your time with my mistake.

P.S. I also made sure to put the folders inside the `~/.local/share/GSC Game World/S.T.A.L.K.E.R. - Clear Sky` folder instead of symlinking them.
